new/LIT7235PPSolomonSmiths.xml

```diff
@@ -0,0 +1,77 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7235PPSolomonSmiths" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Prayer by the virtue of the names by which Solomon was saved from the blacksmiths</title>
+                <title xml:lang="gez" xml:id="t2">ድኅነ፡ ሰሎሞን፡ እምእዴሆሙ፡ ለነሀብት፡</title>
+                <title xml:lang="gez" corresp="#t2" type="normalized">Dǝḫna Salomon ʾǝmʾǝdehu la-nahabt</title>
+                <title xml:lang="en" corresp="#t2">Solomon was saved from the hand of the blacksmiths</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="BLorient4943"/>
```

Comment:
Since you are working on the ms record, I do not think you need to list it here. This witness is not used in the edition and  you are encoding the manuscript description yourself. 
the guidelines says: If manuscripts are known to contain the work but have not been properly encoded yet it is recommended to list them as <witness>[↗](https://betamasaheft.eu/Guidelines/?id=witness)

Reply:
(@nafisa-valieva , I have edited your comment to mark up the marked up parts so that they become visible)
![image](https://github.com/user-attachments/assets/9e356042-e801-4715-9edd-c56dc2962225)

Reply:
thank you!